### ISSUE TITLE
Location box should scroll when large

### ISSUE THREAD

**Issue by jaredkhan** [OneZoom/OZtree#930]
A recent regression I think, works fine on onezoom.org, but now deep locations run off the top of the page

<img width="934" alt="image" src="https://github.com/user-attachments/assets/e300ff9d-13c0-4ece-bbbe-7726cbe7f3ba">

**Comment by jaredkhan** [comment]
fyi @lentinj I think this might have popped up during #928

**Comment by lentinj** [comment]
Yeah, that seems incredibly likely---serves me right for thinking there was a simple fix.
